### PR TITLE
Move Send Search to home screen

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/608?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#608** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/608?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#607** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/607?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/home/screen.tsx [hunk 1]
```diff
@@ -20,160 +25,201 @@ import { TokenBalanceCard } from './TokenBalanceCard'
...
+            ai={'center'}
+            gap="$5"
+            jc="space-around"
+            w={'100%'}
+          >
+            <XStack w="100%">
+              <DepositPopover />
+            </XStack>
+            <XStack ai="center">
+              <Paragraph>Or direct deposit on base</Paragraph>
```
Comment (0xBigBoss) [inline packages/app/features/home/screen.tsx:110]:
```suggestion
              <Paragraph>Or, direct deposit on Base</Paragraph>
```

File: packages/playwright/tests/send.onboarded.spec.ts [hunk 2]
```diff
@@ -93,6 +93,18 @@ for (const token of tokens) {
       assert(!!profile.name, 'profile name not found')
       assert(!!profile.sendId, 'profile send id not found')
       assert(!!plan.sendAccounts[0], 'send account not found')
+
+      // fund account gas fees
```
Comment (0xBigBoss) [inline packages/playwright/tests/send.onboarded.spec.ts:97]: this shouldn't be required. it's set here https://github.com/0xsend/sendapp/blob/7413191665a488db5054c925ff16506e04496c53/packages/playwright/tests/send.onboarded.spec.ts#L248

Comment (youngkidwarrior) [reply packages/playwright/tests/send.onboarded.spec.ts:97]: Account needs gas to see search bar

Comment (0xBigBoss) [reply packages/playwright/tests/send.onboarded.spec.ts:97]: Hmm, they should already have USDC, it's set here as part of the fixture. 

https://github.com/0xsend/sendapp/blob/7413191665a488db5054c925ff16506e04496c53/packages/playwright/tests/fixtures/send-accounts/test.ts#L59

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jul 19, 7:54 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/608).

**Pull request merged**